pom.xml

```diff
@@ -48,4 +81,77 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <profiles>
+    <profile>
+      <id>release</id>
+
+      <distributionManagement>
+        <repository>
+          <id>bintray</id>
+          <url>https://api.bintray.com/maven/opentracing/maven/opentracing-java/;publish=1</url>
```

Comment:
Shouldn't this be `https://api.bintray.com/maven/opentracing/maven/java-jaxrs/;publish=1` ?

**Note:** I'm definately no expert on this; my release actually isn't working yet either: https://github.com/opentracing-contrib/java-globaltracer/issues/2